common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -811,21 +822,15 @@
         "semi": [
             "off"
         ],
-        "semi-spacing": [
-            "error"
```

Comment:
Should make sure semi-spacing continues to error

Reply:
Switched to error.